guides/source/layout.html.erb

```diff
@@ -62,7 +71,7 @@
         <ul class="nav">
           <li><a class="nav-item" id="home_nav" href="https://rubyonrails.org/">Home</a></li>
           <li class="guides-index guides-index-large">
-            <a href="index.html" id="guidesMenu" class="guides-index-item nav-item">Guides Index</a>
+            <a href="index.html" id="guidesMenu" aria-controls="guides" aria-expanded="false" class="guides-index-item nav-item">Guides Index</a>
```

Comment:
What do you think about giving a button role to this link? It's not really navigating to any page, and that's the only purpose of a link actually. (Unless JavaScript is disabled, in which case it shouldn't have those other ARIA attributes either if that's the case).

```suggestion
            <a href="index.html" id="guidesMenu" role="button" aria-controls="guides" aria-expanded="false" class="guides-index-item nav-item">Guides Index</a>
```

Reply:
I like it. 

Let's apply it in JavaScript so it's only a button if... It's a button 😁